File: Leetcode_2678.java
public class Solution {
    public int countSeniors(String[] details) {
        int count = 0;
        for (int i = 0; i < details.length; i++) {
            char[] temp = details[i].toCharArray(); // convert string to char array
            int age = Integer.parseInt("" + temp[11] + temp[12]); // combine chars and parse int
            if (age >= 60) {
                count++;
            }
        }
        return count;
    }

File: Leetcode_2678.py


class Solution:
    def countSeniors(self, details: List[str]) -> int:
        count = 0
        for i in range(len(details)):
            temp = list(details[i])
            age = int(temp[11] + temp[12])
            if age > 60:
                count += 1
        return count
